### ISSUE TITLE
Scripts to analyze .dec files

### ISSUE THREAD

**Issue by manuelfs** [umd-lhcb/lhcb-ntuples-gen#45]
@yipengsun we need a couple of fairly straightforward scripts to more easily study the `.dec` files that probably won't take you long. They would allow us at a glance to see which particles are decayed, with which FFs and BFs with the file, and you will gain some familiarity with the configuration of the MC we will be producing
- A script that takes as input a `.dec` file and outputs only the decays in the file, with their branching fractions normalized to 1 with 4 decimals. For instance, the beginning of the output for [11873010](https://gitlab.cern.ch/lhcb-datapkg/Gen/DecFiles/-/blob/v30r46/dkfiles/Bd_Dststmunu,D0=cocktail,TightCuts.dec) would be
```
Decay B0sig 
  0.2970  MyD_0*-    mu+  nu_mu         PHOTOS  ISGW2;
  0.3069  MyD'_1-    mu+  nu_mu         PHOTOS  ISGW2;
  0.2772  MyD_1-     mu+  nu_mu         PHOTOS  ISGW2;
  0.1188  MyD_2*-    mu+  nu_mu         PHOTOS  ISGW2;
Decay Mytau-
  1.0000   mu-   nu_tau    anti-nu_mu        PHOTOS  TAULNUNU;
Decay MyD0
  1.0000   K-  pi+                              PHOTOS PHSP;
Decay MyD*0
  0.6190   MyD0  pi0                            PHOTOS VSS;
  0.3810   MyD0  gamma                          PHOTOS VSP_PWAVE;
...
```
- A script that downloads the `.dec` files taking the MC type, and matching it to the gitlab link with this [site](http://lhcbdoc.web.cern.ch/lhcbdoc/decfiles/releases/latest/table_evttype.php). To download the file it would simply be
```
get_dec_file.py 11873010
```
Could you do the first by tomorrow's meeting?

**Comment by yipengsun** [comment]
I plan to utilize this tool: https://github.com/scikit-hep/decaylanguage

BTW, I think it makes more sense if we do this in a separate file, because dec file is more of less separated from our ntuple generation workflow, and if we use a separate repo, it can be easier for people to use the project (e.g. They don't need to clone the large `lhcb-ntuples-gen` repo to start using dec file related tools).

**Comment by yipengsun** [comment]
I think what you want is basically a picture like this for each .dec file:
![sample decay](https://raw.githubusercontent.com/scikit-hep/decaylanguage/master/images/DecayChain_Dst_stable-D0-and-D+.png)

**Comment by yipengsun** [comment]
And we can achieve that in a Jupyter notebook.

**Comment by manuelfs** [comment]
Indeed, choose the repo of your liking. I definitely want the text output I requested with the same format as the dec file and the FFs, but that image is also very cool!

As usual, try to minimize dependencies and only add them if they are necessary. Thank you.
